### ISSUE TITLE
loss (Loss 0.00000) for all along the training 

### ISSUE THREAD

**Issue by Ihebzayen** [SeanNaren/deepspeech.pytorch#681]
THE LOSS is equal to zero after some iterations 

![Screenshot from 2022-05-28 18-17-28](https://user-images.githubusercontent.com/88541984/170835942-74b86c57-9970-43ee-985e-1c1d99beeb60.png)

can anyone help me and thanks in advance

**Comment by Ihebzayen** [comment]
any help please !

**Comment by Themba4Sho** [comment]
Try lowering your learning rate; maybe check if your input data is not 'corrupt'.

**Comment by farisalasmary** [comment]
@Ihebzayen make sure that the output length <= the input length
